docs/source/conf-schemas.rst

```diff
@@ -290,6 +294,36 @@ To extract these nested records, use the ``configuration`` option ``json_path``:
     }
   }
 
+Log Patterns
+~~~~~~~~~~~~
+
+Log patterns provide the ability to differentiate log schemas that are identical or very close in nature. They can be added by
```

Comment:
Can you make the first sentence more specific on how that works? Also, pls add some line breaks instead of a big paragraph (it's easier to read)